src/components/EmptyCharacterSheetSlot/index.tsx

```diff
@@ -96,8 +96,8 @@ const EmptySlotTypeIcons = ({ slotTypes }: { slotTypes: CardSlotMetadata["slotTy
                 height: "42px",
                 width: "42px",
                 opacity: "70%",
-                borderRadius: "8px",
-                border: "1px solid black",
+                borderRadius: "20px",
```

Comment:
2px i think.

Reply:
Have made this consistent with the EditableStat radius + box shadow styles